### DISCUSSION TITLE
Unknown symbols on Windows

### DISCUSSION THREAD

**Discussion by gleb498** [daniele77/cli#191] [Q&A]
Hello,

I try to input commands on windows PowerShell but get those strange symbols instead. What could be the problem?

![image](https://user-images.githubusercontent.com/20151819/225310708-2f0602a3-b14a-41c8-aa36-dc7ffe46cc5a.png)

**Comment by daniele77** [comment]
Hi @gleb498 .
Have you tried running the example `complete` that comes with the library?
